crates/revm/src/db/states/bundle_state.rs

```diff
@@ -363,9 +374,12 @@ impl BundleState {
         self.reverts.extend(other.reverts);
     }
 
+    // mattsse: It is weird to me that this returns type Self, but only includes the reverts
+    // should this return only the reverts?
```

Comment:
Yeah, it was wierd, made a change here: https://github.com/bluealloy/revm/pull/681
and now return the `Reverts`.